docs/get-started/tutorials/tutorial-hdfs-logs-distributed-search-aws-s3.md

```diff
@@ -244,49 +271,6 @@ Now that you have a search cluster, ideally, you will want to load balance exter
 This can quickly be done by adding an AWS load balancer to listen to incoming HTTP or HTTPS traffic and forward it to a target group.
 You can now play with your cluster, kill processes randomly, add/remove new instances, and keep calm.
 
-
```

Comment:
Can you open an issue to later add a better example on time-based pruning?

Reply:
https://github.com/quickwit-oss/quickwit/issues/3985